### DISCUSSION TITLE
Room leaders feedback

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/VC-Community-Docs#134] [Coffees]
We didn't have much time for room leaders, feedback, so let's try it here:

1. What's going well?
2. what’s not?
3. what can be improved?
4. what can be added to the process?
5. what can be removed from the process?

**Comment by kierblk** [comment]
Here are just a few thoughts.... 

1. Going Well
- Clearly defined processes help meetings go well
- Clearly defined processes help new members onboard well

2. NOT Going Well
- 

3. Needs Improvement
-

4. Things to Try
-  Can we plan VC notetakers and room leaders earlier... Maybe plan the week on Monday?
- In the spirit of a consistent and awesome experience for all, can giving and receiving feedback (the good and the bad) for room leaders happen? I always want to get better.

5. Things to Retire
-

**Reply by kierblk** [reply]
I left placeholders for each category in case I thought of something and wanted to add later.

**Reply by BekahHW** [reply]
This is great, Sara. I am very on board with your things to try.

**Comment by tomcudd** [comment]
1. Going Well
- Very clearly defined outline for leaders to follow allowing focus on content and moving conversation along
- Very simple process to get a new room leader going
- People willing to jump in and cover as backup or extra room person
- The questions prepared ahead of time is so helpful and awesome
- Each VC goes on like a well oiled machine to get right into the good conversations
2. NOT Going Well
- Worried for Bekah working so hard, glad she's taking a vacation so I can stop being worried for her
3. Needs Improvement
- Need to find ways for quiet, less assertive folks to continue or contribute conversation
4. Things to Try
- Ad hoc/off schedule VC meetings? Concerned that day job people with less flexibility can't join or some international timezones excluded.
- Maybe find a way to select the type of room i.e. intro/ambi/extro-vert options? That might be quite a challenge to organize via zoom.
5. Things to Retire
- What is a soup?

**Comment by MikeRogers0** [comment]
1. What's going well?

I really enjoy the energy boosts I get from talking to everyone. Most weeks I'm quite excited to see what everyone had been up to & I feel like I've made some amazing friends though this amazing community.

2. what’s not?

Sometimes I notice people attend once, but never end up in the slack.

3. what can be improved?

Follow up from 2, I think a few newbies get a bit confused. I suspect the description on the meetingpoint page might need some more detail added to it.

Custom backgrounds for CV Leaders/note takers - I think it can make them stand out more? IDK.

More Glen.

4. what can be added to the process?

I've started to enjoy the non-coffee calls a lot more. I think the adhoc ness is what makes it special. Maybe something to announce them more could be fun.

5. what can be removed from the process?

**Comment by tkshill** [comment]
This is great feedback so far :)

**Comment by BekahHW** [comment]
Thanks to everyone who's responded. Here are my thoughts:

1. What's going well?
     - We support a lot of people
     - There's been a variety of conversations lately
     - I love being able to learn from so many different people
     - I like having the backpocket topics, but I never use them lol
2. what’s not?
    - 
3. what can be improved?
    - I'd like to add more people as hosts
4. what can be added to the process?
     - Definitely agree with Sara that we need to be open to or incorporate feedback in some ways.
     - I wonder if it makes sense to have a session for new folks every once and a while. Like an extra onboarding or a time when new friends can meet some friends who have been here for a while.
     - I also wonder if we should offer some training for new room leaders or support. 
     - I'd also love to make sure we can all experience each others room leading. 
5. what can be removed from the process?

**Reply by tomcudd** [reply]
- I like a "new here" session every so often (once a month? quarter?) for people who have recently joined.
- I would love feedback on the leading I do.
- I would love some training for new prospective room leads, maybe shadowing, or having a seasoned person co-lead

**Comment by bogdaaamn** [comment]
- **What's going well?**
   - We're all great :) 
   - I really love that people feel safe and confident enough to ask questions during VCs
   - I think all the volunteers do a great job spontaneously when last minute changes are happening 

- **What’s not?**
   - Always afraid I might scare people away if they end up in my room for the their first coffee 😅

- **What can be improved?**
    - The docs and conversations we have (on the slack channel) are enough, but I think a space/process where all the volunteers might share feedback, but also experiences related to leading/notetaking would be nice. _eg_ the call we had two weeks ago

- **What can be added to the process?**
   - Virtual Coffee: _The Musical_

- **What can be removed from the process?**
    - //

**Comment by tkshill** [comment]
Seconding Bekah's question. Would folks want/appreciate feedback/training about room leading?

**Reply by SuzeShardlow** [reply]
@tkshill I received some unsolicited positive feedback the other day.

After my first time room hosting, a member reached to me via Slack DM with praise 😳 This made me happy 😀

I'm always keen to know what people like and don't like so I would welcome more feedback on how I'm doing.

**Reply by tkshill** [reply]
@SuzeShardlow that's wonderful! I've gotten some surprise good feedback as well and it totally made my day. So I definitely see why we could do that more!

**Reply by MikeRogers0** [reply]
If I have a good room, I always try to say thank you to the key people afterwards. Like, I've had rooms where I thought I done a terrible job & then realised it was total imposter syndrome after someone said I did good.

**Comment by BekahHW** [comment]
Ok, some action items I see here are:|

1. More opportunities or systems for feedback
2. Better onboarding and/or training for new room leaders
3. Opportunities for feedback
4. Create clearer processes, explanations, and maybe quarterly sessions for new members
5. Work on ways to make the room friendlier to all attendees
